views/index.erb

```diff
@@ -39,7 +39,7 @@
 
         <span
           class='workos-container'
-          data-prop-app-name='SaaS Co.'
+          data-prop-app-name='Enterprise Co.'
```

Comment:
We probably don't want this to refer to the Enterprise since it's the name of the Developer's application that will be revealed to the Enterprise IT Admin.

Reply:
Ohhh got it, I'll fix that